ntoskrnl/io/iomgr/deviface.c

```diff
@@ -412,6 +632,182 @@ IopOpenInterfaceKey(IN CONST GUID *InterfaceClassGuid,
     return Status;
 }
 
+/**
+ * @brief IoGetDeviceInterfaceAlias
+ *
+ * Returns the alias device interface of the specified device interface
+ * instance, if the alias exists.
+ *
+ * @param[in] SymbolicLinkName
+ * Pointer to a symbolic link string which identifies the device interface instance.
+ *
+ * @param[in] AliasInterfaceClassGuid
+ * Pointer to a device interface class GUID.
+ *
+ * @param[out] AliasSymbolicLinkName
+ * Pointer to unicode string which receives the alias symbolic link upon success.
+ * Must be freed with RtlFreeUnicodeString after using.
+ *
+ * @return NTSTATUS values in case of errors, STATUS_SUCCESS otherwise.
+ *
+ * @remarks Must be called at IRQL = PASSIVE_LEVEL in the context of a system thread
+ *
+ */
+NTSTATUS
+NTAPI
+IoGetDeviceInterfaceAlias(
+    _In_ PUNICODE_STRING SymbolicLinkName,
+    _In_ CONST GUID *AliasInterfaceClassGuid,
+    _Out_ PUNICODE_STRING AliasSymbolicLinkName)
+{
+    UNICODE_STRING AliasSymbolicLink;
+    UNICODE_STRING PrefixString;
+    UNICODE_STRING AliasGuidString;
+    UNICODE_STRING DeviceString, ReferenceString;
+    PKEY_VALUE_FULL_INFORMATION kvInfo;
+    HANDLE DeviceKey, AliasInstanceKey;
+    PVOID Buffer;
+    USHORT i;
+    NTSTATUS Status;
+
+    RtlInitEmptyUnicodeString(&PrefixString, NULL, 0);
+    RtlInitEmptyUnicodeString(&DeviceString, NULL, 0);
+    RtlInitEmptyUnicodeString(&ReferenceString, NULL, 0);
+    RtlInitEmptyUnicodeString(&AliasGuidString, NULL, 0);
+    RtlInitEmptyUnicodeString(&AliasSymbolicLink, NULL, 0);
```

Comment:
You could instead do, for example, `UNICODE_STRING PrefixString = {0};`, etc.

Reply:
Done. Changed the style of the initialization of these strings.